package.json

```diff
@@ -40,6 +40,7 @@
     "react-redux": "^8.0.5",
     "swr": "^2.0.0",
     "usehooks-ts": "^2.9.1",
+    "yarn": "^1.22.19",
```

Comment:
Hey I think you've installed your yarn package manager without using --global flag so it installed in our dependencies, please follow https://classic.yarnpkg.com/en/docs/install to install yarn. If you didn't install any packages there should be only SubmitPageForm.tsx that changed.

Reply:
thanks for pointing this @JakubChorzepa

Reply:
Hi. Thank you for the heads up. I did not notice this. I will correct this mistake.

And that is exactly what happened. I did not know I had to install it globally. I appreciate the information.

Reply:
No problem, if you need any help you can join our discord server and me or someone else will surely help you.